auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -51,7 +51,8 @@
 
       <!-- Type -->
       <template v-slot:item-slot-Type="{ item }">
-        <div class="gray-9 font-weight-bold">{{ type(item) }}</div>
+        <div class="gray-9 font-weight-bold type-text">{{ type(item) }}</div>
+        <div v-if="enableNameRequestType && isNameRequest(item)" class="gray-9 font-weight-bold type-text ml-1">{{ nameRequestType(item) }}</div>
         <div>{{ typeDescription(item) }}</div>
       </template>
 
```

Comment:
Adding the NR type in a separate `div` is a little weird here, but it's to avoid an issue with the Type filter. When the NR type is included in the same `div`, the filter treats each unique NR type as a new item to filter on, resulting in the screenshot below. Notice "Name Request - Incorporation" and "Name Request - Registration". Separating them into different `div`s is a (not great) workaround. If anyone has experience with `v-data-table` filters and knows a better fix, let me know.
 
![Screenshot 2023-07-18 073104](https://github.com/bcgov/sbc-auth/assets/122323255/d632e00e-aa3a-401b-887d-eba1771ebdd6)

Reply:
How does this look?

Did you look at affiliations-factory.ts::updateFilter() ? Maybe you could tweak that to match all Name Requests regardless of entity type ...?

Reply:
I'm not sure what filtering looks at, either, but your solution works:

![image](https://github.com/bcgov/sbc-auth/assets/10002889/01408da8-da0a-495d-bb54-629159d3504f)

Reply:
I just had a look at `affiliations-factory.ts::updateFilter()` and it doesn't seem to get called when the popup appears, it only gets called once the Type is selected. The problem must be happening at some point before that, or as part of the vuetify component itself.

Reply:
I think it's OK as you have it, but a code comment would be ideal.

Reply:
I'm OK to resolve this conversation.